go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/ramr/go-reaper
+
+go 1.16
```

Comment:
1.16 is wee bit too recent. Maybe set it a few years back ala 1.12?  Thx

Reply:
That version actually does not do anything for older golang versions.
And I am pretty sure that code would still compile for older versions.

Reply:
How about a bit more recent that 1.12, something like 1.14?

Reply:
Sure that works. Thx

Reply:
Actually don't bother changing it. Have a few spare cycles now (before dinner!!). 
I'll merge it and lower the version.

Reply:
Thanks!